action.yml

```diff
@@ -1,18 +1,46 @@
-name: 'The name of your action here'
-description: 'Provide a description here'
-author: 'Your name or organization here'
+name: "Make state repos' dispatches"
+description: 'Dispatch image changes on multiple state repos'
+author: 'Prefapp'
 
 # Define your inputs here.
 inputs:
-  milliseconds:
-    description: 'Your input description here'
+  dispatches_file:
+    description: 'Path to the make_dispatches.yaml file of your repo'
     required: true
-    default: '1000'
-
-# Define your outputs here.
-outputs:
-  time:
-    description: 'Your output description here'
+    default: '.github/make_dispatches.yaml'
+  token:
+    description: Github token
+    required: false
+    default: ${{ github.token }}
+  dispatch_type:
+    description: Dispatch types (releases, snapshots, *)
+    required: false
+    default: '*'
+  flavors:
+    description: Flavors to dispatch separated by comma
+    required: false
+    default: default
+  state_repo:
+    description:
+      List of state repos to dispatch separated by comma (use "*" to target all
+      flavors)
+    required: false
+    default: '*'
+  default_releases_registry:
+    description: Default registry to use when dispatching releases
+    required: false
+  default_snapshots_registry:
+    description: Default registry to use when dispatching snapshots
+    required: false
+  reviewers:
+    description: List of reviewers' login username, separated by commas
+    required: false
+    default: ''
+  origin_repo:
+    description:
+      Path (owner/repo_name) of the repo that originally called this action
+    required: true
+    default: ''
```

Comment:
Can't be inferred from the github context? @juanjosevazquezgil please take a look to https://github.com/actions/toolkit/tree/main/packages/github